spec/factories/line_items.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :line_item do
+    cart_id 1
```

Comment:
Why the explicit ids? It's a bit unusual to have ids in your factories, and associations like this are usually set up more like:
```
factory :line_item do
  cart
  book
 quantity 1 # or rand(1..4) or something like that
end
```
docs here: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#associations